.github/workflows/node-version-up.yml

```diff
@@ -0,0 +1,63 @@
+name: Node Version Up
+
+on:
+  workflow_dispatch:
+    inputs:
+      versioning:
+        description: "node version up: major, minor, or patch"
+        required: true
+        default: "patch"
+
+jobs:
+  bump-version:
+    runs-on: ubuntu-latest
+    name: bump version
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Setup User
+        run: |
+          git config user.name github-actions
+          git config user.email github-actions@github.com
+      - name: install cargo-edit
+        run: |
+          cargo install cargo-edit
+      - name: Set New Version ${{ github.event.inputs.versioning }}
+        continue-on-error: false
+        run: |
+          cargo set-version --bump ${{ github.event.inputs.versioning }} --package pendulum-node
```

Comment:
I would prefer to change the version of all crates but this is to be seen after https://github.com/pendulum-chain/pendulum/pull/493.

Reply:
I've added 3 choices: 
node, all packages, or all packages but the node.

Reply:
I think we don't need to be able to choose between the three. It's nice-to-have but maybe better to remove since you had to duplicate large parts of it.

Reply:
So how do you want to upgrade? Just between node or ALL?

Reply:
Let's update all crates by default.

Reply:
default set to true.